src/components/ViewRequest/Actions/SendRequest.tsx

```diff
@@ -97,6 +103,20 @@ const SendRequest = () => {
     }
   };
 
+  const draftPackageHandler = () => {
```

Comment:
Any reason for not putting this in the updateHandler, like the others?  Understand breaking out from a code readability perspective, but maybe should put the functions to determine next stage and text in a reusable function itself, and use it here rather than hardcoding -- especially b/c I don't believe the subStage defined is a valid one, and if things change it's in one spot, not possibly multiple.